alf/utils/value_ops.py

```diff
@@ -180,101 +246,97 @@ def discounted_return(rewards, values, step_types, discounts, time_major=True):
     return rets.detach()
 
 
-def one_step_discounted_return(rewards, values, step_types, discounts):
-    """Calculate the one step discounted return  for the first T-1 steps.
+def first_step_future_discounted_returns(rewards,
```

Comment:
This ``first_step_future_discounted_returns`` function is the only new thing in this file.